dist/system-csp.src.js

```diff
@@ -1151,13 +1151,11 @@ function global(loader) {
     }
 
     loader.set('@@global-helpers', loader.newModule({
-      prepareGlobal: function(moduleName, deps, exportName) {
-        // first, we add all the dependency modules to the global
-        for (var i = 0; i < deps.length; i++) {
-          var moduleGlobal = moduleGlobals[deps[i]];
-          if (moduleGlobal)
-            for (var m in moduleGlobal)
-              loader.global[m] = moduleGlobal[m];
+      prepareGlobal: function(moduleName, exportName, globals) {
```

Comment:
I don't think we can make breaking changes to the @@global-helpers API. It's a somewhat open API, transpile uses for example and this change would break it: https://github.com/stealjs/transpile/blob/master/lib/global_amd.js#L26